README.md

```diff
@@ -121,18 +121,17 @@ MySQL.escape(conn::MySQL.Connection, str::String) -> String
 ```
 Escape an SQL statement
 
-#### MySQL.query
+#### MySQL.Query (previously MySQL.query)
 
 ```julia
-MySQL.query(conn::MySQL.Connection, sql::String, sink=Data.Table; append::Bool=false) => sink
+MySQL.Query(conn::MySQL.Connection, sql::String; append::Bool=false) => sink
 ```
-Execute an SQL statement and return the results in the `sink`, which can be any valid `Data.Sink` (interface from [DataStreams.jl](https://github.com/JuliaData/DataStreams.jl)). By default, a NamedTuple of Vectors is returned.
+Execute an SQL statement and return the results as a MySQL.Query object (see [MySQL.Query](#mysqlquery)).
 
-Passing `append=true` as a keyword argument will cause the resultset to be _appended_ to the sink instead of replacing.
-
-To get the results as a `DataFrame`, you can just do `MySQL.query(conn, sql, DataFrame)`.
+The results can be materialized as a data sink that implements the Tables.jl interface.
+E.g. `MySQL.Query(conn, sql) |> DataFrame` or `MySQL.Query(conn, sql) |> columntable`
 
-See list of DataStreams implementations [here](https://github.com/JuliaData/DataStreams.jl#list-of-known-implementations)
+Passing `append=true` as a keyword argument will cause the resultset to be _appended_ to the sink instead of replacing.

```

Comment:
Hmmm, this is no longer applicable. Each sink provides it's own way of appending.

Reply:
Removed that line